src/test/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImplTest.java

```diff
@@ -150,4 +150,14 @@ void shouldSuccessfullyFindFinancialTransaction_WhenFindingWithProperTransaction
         //then
         Assertions.assertEquals(financialTransactionDTO, foundTransaction);
     }
+
+    @Test
```

Comment:
Please add correct @DisplayName for test.

Reply:
Okay, done